Violet Rails.xcodeproj/project.pbxproj

```diff
@@ -17,6 +17,8 @@
 		3E2A29172868944800585E9E /* Violet_RailsUITests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3E2A29162868944800585E9E /* Violet_RailsUITests.swift */; };
```

Comment:
@josessolmaximo is this a compiled file? If this can be ignored (.gitignore)  it would be great because it doesn't look human readable

Reply:
I think this file contains the structure of the project so it can't be ignored